sormas-backend/src/test/java/de/symeda/sormas/backend/region/ContinentFacadeEjbTest.java

```diff
@@ -43,14 +44,14 @@ public void testSubcontinent() {
 		assertEquals(1, subcontinents.size());
 		final SubcontinentDto savedSubcontinentDto = getSubcontinentFacade().getByUuid(subcontinents.get(0).getUuid());
 		assertEquals("CENTRAL_EUROPE", savedSubcontinentDto.getDefaultName());
-		assertEquals("EUROPE", savedSubcontinentDto.getContinent().getCaption());
+		assertEquals(I18nProperties.getContinentName("EUROPE"), savedSubcontinentDto.getContinent().getCaption());
 
 		final List<SubcontinentIndexDto> indexList =
 			getSubcontinentFacade().getIndexList(new SubcontinentCriteria().continent(continent.toReference()), null, null, null);
 		assertEquals(1, indexList.size());
 		final SubcontinentIndexDto subcontinentIndexDto = indexList.get(0);
 		assertEquals("CENTRAL_EUROPE", subcontinentIndexDto.getDefaultName());
-		assertEquals("EUROPE", subcontinentIndexDto.getContinent().getCaption());
+		assertEquals(I18nProperties.getContinentName("EUROPE"), subcontinentIndexDto.getContinent().getCaption());
 		assertEquals("Central Europe", subcontinentIndexDto.getDisplayName());
```

Comment:
This does not cover the problematic cases (continent unknown or default name containing brackets). 
Please add tests for these cases. Also, better add a new test to specifically test the functionality of `getContinentName`, `getSubcontinentName` and the cleanup method.

Reply:
That has been done in my changes to `I18nPropertiesTest.java`. The code above just fixed a failing test

Reply:
Right, sorry :-)